mysqlcluster/mysqlcluster.go

```diff
@@ -270,6 +270,22 @@ func (c *MysqlCluster) EnsureVolumes() []corev1.Volume {
 			},
 		})
 	}
+	// add the ssl secret mounts
```

Comment:
comments ： Upcase the first letter, and add dot after statement compeleted.
-> `Add the ssl secret mounts.`